bundles/org.openhab.binding.solarforecast/pom.xml

```
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.solarforecast</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: SolarForecast Binding</name>
+  <dependencies>
+    <!-- version needs to match with other projects like org.openhab.io.openhabcloud.pom.xml -->
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20180813</version>
```

It's vulnerable, see CVE-2022-45688

```suggestion
      <version>20230618</version>
```

---

Done